src/lib.rs

```diff
@@ -42,7 +43,7 @@ pub struct DetectionOptions {
 impl Default for DetectionOptions {
     fn default() -> Self {
         DetectionOptions {
-            fast_analysis: false,
+            fast_analysis: SceneDetectionSpeed::Fast,
```

Comment:
Currently the default is to use slow analysis, so I'd prefer to keep that as the default.